src/Parser/Rogue/Assassination/CHANGELOG.js

```diff
@@ -1,9 +1,9 @@
-import { tsabo } from 'MAINTAINERS';
+import { Cloake } from 'MAINTAINERS';
 
 export default  [
   {
-    date: new Date('2018-01-20'),
-    changes: 'Set up the spec in a way to make it easy for someone else to contribute. Added a few basic metrics based on my current Subtlety implementation.',
-    contributors: [tsabo],
+    date: new Date('2018-02-12'),
```

Comment:
Don't delete old changelogs.

Reply:
Sorry about that! What's the appropriate way to update the changelog and maintainer status?

Reply:
A nice touch would be to use `ItemLink` for the Duskwalker's Footpads
```jsx
import ITEMS from 'common/ITEMS';
import ItemLink from 'common/ItemLink';
import Wrapper from 'common/Wrapper';
....
changes: <Wrapper>Added support for <ItemLink id={ITEMS.DUSKWALKERS_FOOTPADS.id} icon />.</Wrapper>,
```